src/analysis/special_functions/gamma.lean

```diff
@@ -337,3 +336,183 @@ end
+  rw [this, integrable_on_union],
+  split,
+  { split,
+    { refine continuous_on.ae_strongly_measurable (continuous_on.mul _ _).norm measurable_set_Ioc,
+      { apply (continuous_exp.comp continuous_neg).continuous_on.mul,
+        apply continuous_on.mono continuous_on_log, simp, },
+      { apply continuous_on_id.rpow_const, intros x hx, right, linarith }, },
+    { apply has_finite_integral_of_bounded,
+      swap, { exact 1 / (s - 1), },
+      refine (ae_restrict_iff' measurable_set_Ioc).mpr (ae_of_all _ (λ x hx, _)),
+      rw [norm_norm, norm_eq_abs, mul_assoc, abs_mul],
+      have : 1/(s-1) = 1 * (1 / (s-1)) := by ring, rw this,
+      refine mul_le_mul _ _ (by apply abs_nonneg) zero_le_one,
+      { rw [abs_of_pos (exp_pos(-x)), exp_le_one_iff, neg_le, neg_zero], exact hx.1.le },
+      { apply le_of_lt, refine abs_log_mul_self_rpow_lt x (s-1) _ (by linarith),
+        rw Ioc at hx, exact hx, }, }, },
+  { have := is_O.norm_left (dGamma_integrand_is_O_at_top s),
+    refine integrable_of_is_O_exp_neg one_half_pos (continuous_on.mul _ _).norm this,
+    { refine (continuous_exp.comp continuous_neg).continuous_on.mul (continuous_on_log.mono _),
+      simp, },
+    { apply continuous_at.continuous_on, intros x hx,
+      apply continuous_at_id.rpow continuous_at_const,
+      dsimp, right, linarith, }, }
+end
+
+/-- A uniform bound for the `s`-derivative of the `Γ` integrand for `s` in vertical strips. -/
+private lemma loc_unif_bound_dGamma_integrand {t : ℂ} {s1 s2 x : ℝ} (ht : s1 ≤ t.re ∧ t.re ≤ s2)
+  (hx : 0 < x) :
+  ∥ dGamma_integrand t x ∥ ≤ (dGamma_integrand_real s1 x) + (dGamma_integrand_real s2 x) :=
+begin
+  by_cases (1 ≤ x),
+  { suffices: ∥ dGamma_integrand t x ∥ ≤ dGamma_integrand_real s2 x, -- case 1 ≤ x
+    { have: 0 ≤ dGamma_integrand_real s1 x := by apply abs_nonneg, linarith, },
+    rw [dGamma_integrand, dGamma_integrand_real, complex.norm_eq_abs, complex.abs_mul, abs_mul,
+      ←complex.of_real_mul, complex.abs_of_real],
+    refine mul_le_mul_of_nonneg_left _ (abs_nonneg _),
+    rw complex.abs_cpow_eq_rpow_re_of_pos hx,
+    refine le_trans _ (le_abs_self _),
+    apply rpow_le_rpow_of_exponent_le h,
+    rw [complex.sub_re, complex.one_re], linarith, },
+  { push_neg at h, -- case x < 1
+    suffices: ∥ dGamma_integrand t x ∥ ≤ dGamma_integrand_real s1 x,
+    { have : 0 ≤ dGamma_integrand_real s2 x := by apply abs_nonneg, linarith, },
+    rw [dGamma_integrand, dGamma_integrand_real, complex.norm_eq_abs, complex.abs_mul, abs_mul,
+      ←complex.of_real_mul, complex.abs_of_real],
+    refine mul_le_mul_of_nonneg_left _ (abs_nonneg _),
+    rw complex.abs_cpow_eq_rpow_re_of_pos hx,
+    refine le_trans _ (le_abs_self _),
+    apply rpow_le_rpow_of_exponent_ge hx h.le,
+    rw [complex.sub_re, complex.one_re], linarith, },
+end
+
+namespace complex
+
+/-- The `Γ` function is complex-differentiable at any `s ∈ ℂ` with `1 < re s`. -/
+theorem differentiable_at_Gamma_integral {s : ℂ} (hs : 1 < s.re) :
```

Comment:
Instead of just stating the differentiability, could you state that `Gamma_integral` has the derivative ... at this point, and then deduce the differentiability? The reason is that someone might need the formula for the derivative some day, and it is is not easily available from the current formulation.

Reply:
Good point, I've added a proof of this.